tests/test_part_image.py

```diff
@@ -124,15 +124,29 @@ def test_partition_alignment(self, bitbake_path, bitbake_variables, latest_part_
         assert(parts_start[2] == parts_end[1])
         assert(parts_start[3] == parts_end[2])
 
+        if bitbake_variables["MENDER_DEVICE_TYPE"] == "qemux86_64":
+            # For both data and swap partitons a default offset is added.
+            # Offset value is equal to alignment.
+            assert(parts_start[4] == parts_start[3] + alignment)
+            assert(parts_start[5] == parts_end[4] + alignment)
```

Comment:
These partitions don't exist on the Mender Yocto image, so I believe these will fail. Or actually not, since Yocto uses `qemux86-64`, not `qemux86_64`. But relying on that is very confusing.

Can you instead make it dependent on the existence of those two partitions? IOW, use two ifs, and do `if len(parts_start) > 4` and `if len(parts_start) > 5`, respectively.